src/ServiceControl/Alerting/Mail/CustomChecksMailNotification.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace ServiceControl.Alerting.Mail
+{
+    using System.Threading.Tasks;
+    using Alerting;
+    using Contracts.CustomChecks;
+    using Infrastructure.DomainEvents;
+    using Raven.Client;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    class CustomChecksMailNotification : IDomainHandler<CustomCheckFailed>, IDomainHandler<CustomCheckSucceeded>
+    {
+        readonly IDocumentStore store;
+        readonly string emailDropFolder;
+
+        public CustomChecksMailNotification(IDocumentStore store, Settings settings)
+        {
+            this.store = store;
+            emailDropFolder = settings.EmailDropFolder;
+        }
+
+        public async Task Handle(CustomCheckFailed domainEvent)
+        {
+            var settings = await LoadSettings().ConfigureAwait(false);
+            if (settings == null || !settings.Email.Enabled)
+            {
+                return;
+            }
+
+            await EmailSender.Send(
+                    settings.Email,
+                "Health check failed",
```

Comment:
Should these mention ServiceControl or is it enough that it will come from a particular address?

Should we put details about the ServiceControl instance here to allow customers with many instances to distinguish them? Alternatively we could ask for a name in the notification settings.

Reply:
I've added the name of the service control instance to the subject, and service name and address to the email body.